Sources/ApolloCodegenLib/Frontend/JavaScriptBridge.swift

```diff
@@ -6,12 +6,13 @@ import JavaScriptCore
 
 /// An errror thrown during JavaScript execution.
```

Comment:
Somehow we both missed this typo 😆 
```suggestion
/// An error thrown during JavaScript execution.
```